tasks/cea-install.js

```diff
@@ -136,6 +158,7 @@ const run = async () => {
 
     // change directory to target directory
     process.chdir(path.resolve(process.cwd(), TARGET_DIR));
+    await createGitIgnore();
```

Comment:
Should this be part of `srcInit`, where the rest of the files are also created?  Might make the most sense conceptually.

Reply:
Problem with that, it needs to change directory before it can write the file.